charts/argo-cd/values.yaml

```diff
@@ -233,7 +233,7 @@ dex:
 
   image:
     repository: ghcr.io/dexidp/dex
-    tag: v2.28.1
+    tag: v2.30.0
```

Comment:
I am unsure if we should update dex to a newer verseion than what's bundled within the official kustomize manifests over there: https://github.com/argoproj/argo-cd/blob/v2.0.5/manifests/base/dex/argocd-dex-server-deployment.yaml

Everyone is free to use a newer one and can override it through values. But the app release is tested with the versions used in the kustomize manifests.
Other opinions @mbevc1 @oliverbaehler @davidkarlsen @seanson ?

Reply:
Hm, yeah that seems sensible approach until upstream doesn't support this combo.